src/components/Button.jsx

```diff
@@ -3,7 +3,7 @@ import Link from "next/link";
 const Button = ({ link, text, target, color = "text-black" }) => {
   return (
     <Link
-      className={` ${color} my-6 duration-300 w-fit rounded-sm md:text-2xl 2xl:text-3xl bg-swim-yellow px-8 py-2 font-bold hover:opacity-50 active:opacity-40 z-10`}
+      className={` ${color} my-3 md:my-6 duration-300 py-[1%] md:py-[2%] px-[6%] md:px-[7.5%] w-fit rounded-sm text-md md:text-2xl 2xl:text-3xl bg-swim-yellow px-8 py-2 font-bold hover:opacity-50 active:opacity-40 z-10`}
```

Comment:
can you use numbers instead of percentages?  (px-1 or py-2...)

Reply:
I have addressed the comments, could you review the changes!